src/test/java/com/plaid/client/integration/InstitutionsGetByIdTest.java

```diff
@@ -51,7 +51,7 @@ public void testSuccessWithIncludeOptionalMetadataTrue() throws Exception {
+    assertNotNull(institution.getLogo());
```

Comment:
Seems like the logo changed - I think it's enough to assert that some value is there rather than the exact base64 data.